libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat

```diff
@@ -84,16 +84,16 @@ PC5 RSSI_ADC ADC1
 PA2 LED0 OUTPUT LOW
 
 # USART1
-PA10 USART1_RX USART1
```

Comment:
I'd really like to find out why these make any difference. It feels like voodoo programming.